pkg/receive/hashring.go

```diff
@@ -276,7 +285,11 @@ func HashringFromConfigWatcher(ctx context.Context, algorithm HashringAlgorithm,
 			if !ok {
 				return errors.New("hashring config watcher stopped unexpectedly")
 			}
-			updates <- newMultiHashring(algorithm, replicationFactor, cfg)
+			h, err := newMultiHashring(algorithm, replicationFactor, cfg)
+			if !ok {
```

Comment:
You probably meant this right?

```suggestion
			if err != nil {
```

Reply:
Oh yeah! Thanks

Reply:
:heavy_check_mark: